cmd/tf_operator/main.go

```diff
@@ -83,6 +85,20 @@ func init() {
 		log.Info("No controller_config_file provided; using empty config.")
```

Comment:
Can you add a check if gRPCServerFilePath is specified and prints a warning it if its supplied in the config file? I think in a future release we will want to make it an error to include in the controllerConfigFile.

Also can you define a new struct ControllerConfigFile that only contains options that should be provided by the file (and not flags) and then [embed](https://golang.org/doc/effective_go.html#embedding) that in the existing ControllerConfig struct so that code referencing existing options don't need to be updated?

Reply:
Yeah it works for me, I will update it.